vision_datasets/common/data_manifest/data_manifest.py

```diff
@@ -199,3 +239,39 @@ def __eq__(self, other) -> bool:
 
     def __len__(self):
         return len(self.images)
+
+
+class MultiImageDatasetManifest(ManifestBase):
+    """
+    Multi-image dataset manifest supporting multi-image sample. Image information except label is encapsulated in ImageDataManifest. Annotation information is encapsulated in MultiImageLabelManifest, 
+    including field 'img_ids' to capture indices in images list, and label_data to capture the label.
+    """
+
+    def __init__(self, images: List[ImageDataManifest], annotations: List[MultiImageLabelManifest], data_type: str, addtional_info={}):
+        """
+
+        Args:
+            images (list): image manifest
```

Comment:
You might want to emphasize the image.labels in ImageDataManifest won't be respected.

Some none checks on it might be helpful as well

Reply:
Added comments and a check